util/msys2_install.sh

```diff
@@ -8,25 +8,26 @@ installflip=false
 util_dir=$(dirname "$0")
 
 echo "Installing dependencies needed for the installation (quazip)"
-pacman --needed -S base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip msys/python3 msys/unzip
+pacman --needed --noconfirm -Sy base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip msys/python3 msys/unzip
 
 source "$dir/win_shared_install.sh"
 
 function install_avr {
     rm -f -r "$avrtools"
-    wget "http://ww1.microchip.com/downloads/en/DeviceDoc/avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip"
+    wget "https://blog.zakkemble.net/download/avr-gcc-8.3.0-x86-mingw.zip"
     echo "Extracting AVR toolchain..."
-	unzip -q avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip
-	mv avr8-gnu-toolchain-win32_x86/ avr8-gnu-toolchain
-    rm __MACOSX -R
-    rm avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip
+    unzip -q -d . avr-gcc-8.3.0-x86-mingw.zip
+    mv avr-gcc-8.3.0-x86-mingw avr8-gnu-toolchain
```

Comment:
The bundled make needs to be deleted, because we should already have one installed from pacman, and it breaks the size check for some reason.

```suggestion
    mv avr-gcc-8.3.0-x86-mingw avr8-gnu-toolchain
    rm avr8-gnu-toolchain/bin/make.exe
```